### ISSUE TITLE
Pokemon select menu in Mobile Battles needs EGG check/appropriate string (party_menu.asm)

### ISSUE THREAD

**Issue by gb-mobile** [pokecrystal-mobile/pokecrystal-mobile-en#18]
When selecting a Pokemon for a Mobile Battle (PokeCenter 2F PokeCom Colosseum), the word 'ABLE' is displayed next to all Pokemon, including EGGs - despite EGGs not being selectable.

There needs to be a check to determine whether a POKEMON is an EGG, and display the words 'NOT ABLE' next to it instead of 'ABLE' if it is.

![image](https://user-images.githubusercontent.com/110418063/196676068-673e5b1c-585b-408b-9f6a-5e100cef4a5e.png) 
↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
![image](https://user-images.githubusercontent.com/110418063/197151632-0b08f80a-6757-46cd-b175-e7c0a319031a.png)

Also the 'CANCEL' option in this menu needs moving 2 tiles left, alongside it's respective cursor.
'CANCEL' is able to be moved by just removing two spaces in the string. (It shares a string with OK), but I couldn't find how the cursor was drawn for it.

TL;DR change this:
![image](https://user-images.githubusercontent.com/110418063/197151493-1f784ec4-304d-4f40-8ee2-b9e31cad040c.png)
to this:
![image](https://user-images.githubusercontent.com/110418063/197151406-e6fdc207-4b0a-4403-b72e-13e7f6a78ba8.png)

**Why?**
The spacing between the two words is supposed to be just 2 tiles (1 of which is to account for the cursor) as shown in the Japanese version:

![image](https://user-images.githubusercontent.com/110418063/197150074-080cad91-dfea-42ae-9858-48cd8c0668b6.png)

Currently the gap between OK and CANCEL has just been expanded to account for its original position, but this would be incorrect (and also looks messier).

**Comment by pokecrystal-mobile** [comment]
Also: Is this the only instance of both the 'OK / CANCEL' options the only instance of a menu like this using both?
Is the placement of CANCEL set like this specifically in the EN version?

![image](https://user-images.githubusercontent.com/110418063/196685489-dd265b00-d08b-4b59-ae64-f09d1a42cc1b.png)


If we use the same space gap as the Japan version is should look like this:

![image](https://user-images.githubusercontent.com/110418063/196685597-4737433b-75ef-4238-8b42-a4a4b9f4ea03.png)


But unsure if this was how this was handled officially.
